client/src/components/geneExpression/infoPanel/connect.ts

```diff
@@ -10,32 +9,21 @@ function useConnect({
   dispatch: AppDispatch;
   activeTab: string;
 }) {
-  const [tabValue, setTabValue] = useState(TabsValue.CellType);
+  const [tabValue, setTabValue] = useState(ActiveTab.CellType);
 
   const handleTabsChange = (
     _: ChangeEvent<Record<string, unknown>>,
     tabsValue: unknown
```

Comment:
we can update the `unknown` to `ActiveTab` to remove the type casting `as ActiveTab` below!

Reply:
<img width="680" alt="Screenshot 2024-07-23 at 2 49 13 PM" src="https://github.com/user-attachments/assets/1e3088f6-8642-426c-a1d7-0680ff736d66">

Reply:
Oops - I forgot to update when I did the cleanup thanks!